evergreen/convert-client-cert-to-pfx.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with an error if any of the commands fail
+
+# Environment variables used as input:
+#   CLIENT_PEM                      Path to mongo -orchestration's client.pem: must be set.
+#   MONGO_X509_CLIENT_P12           filename for client certificate in p12 format
+#
+# Enviornment variables produced as output:
+#   MONGODB_X509_CLIENT_P12_PATH    Absolute path to client certificate in p12 format
+#   MONGO_X509_CLIENT_P12_PASSWORD  password for client certificate
```

Comment:
Alternative names could be `MONGO_X509_CLIENT_CERTIFICATE_PATH` and `MONGO_X509_CLIENT_CERTIFICATE_PASSWORD`